site/docs/pages/index.mdx

```diff
@@ -194,7 +270,7 @@ setFilteredTokens(filteredTokens);
 <hr className="my-8 md:my-20" />
 
 <section>
-  <main className="flex flex-row gap-1">
+  <main className="flex flex-row gap-1 font-normal">
```

Comment:
Updating the weight of all major titles on the landing page to resemble the newly added `Everything you need to build onchain` section and Figma mockups.

Reply:
ditto

Reply:
and for the once below